### PR TITLE
feat: aztec-packages

### PR DESCRIPTION
**Pull request by AztecBot**
Development from Aztec.

### CODE REVIEW

**Comment by ludamad** [comment]
@kevaundray we want this branch protected I think so it doesn't get closed every merge

**Review by kevaundray — APPROVED** [review]

File: scripts/install_wasm-bindgen.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -eu
+
+cd $(dirname "$0")/..
+
+# Install wasm-bindgen-cli.
+if [ "$(wasm-bindgen --version | cut -d' ' -f2)" != "0.2.86" ]; then
+  echo "Building wasm-bindgen..."
+  RUSTFLAGS="-Ctarget-feature=-crt-static" cargo install -f wasm-bindgen-cli --version 0.2.86
```
Comment (kevaundray) [inline scripts/install_wasm-bindgen.sh:9]: For reference: this is here because of https://github.com/rust-lang/rust/issues/115450

**Review by kevaundray — APPROVED** [review]

**Comment by kevaundray** [comment]
I don't think we can keep the PR open? We can probably have a CI workflow that creates a new PR like release-please whenever this one gets closed though

**Comment by ludamad** [comment]
Sorry misspoke - so it doesn't get *deleted* every merge

**Pull request merged**